src/Player/PlayerProxy.h

```diff
@@ -45,6 +84,18 @@ class PlayerProxy
      */
     void setOnClose(const std::function<void(const toolkit::SockException &ex)> &cb);
 
+    /**
+    * 在连接尝试时设置主动断开回调
+    * @param cb 回调对象
+    */
+    void setOnDisconnect(const std::function<void()> &cb);
+
+    /**
+    * 在连接尝试时设置活动连接成功回调
```

Comment:
If you prefer English annotations, I think English annotations are better, otherwise machine translation may not be accurate.